taichi/backends/dx/dx_api.h

```diff
@@ -0,0 +1,43 @@
+#pragma comment (lib, "d3d11.lib")
+#pragma comment (lib, "d3dcompiler.lib")
+#pragma comment (lib, "dxguid.lib")
+
+#include "taichi/common/core.h"
+#include "taichi/backends/device.h"
+#include <d3d11.h>
+#include <d3dcompiler.h>
+
+TLANG_NAMESPACE_BEGIN
+
+namespace directx11 {
+
+bool initialize_dx(bool error_tolerance = false);
```

Comment:
is `error_tolerance` used here?

Reply:
Ah ..! It is not used and is now removed. (actually initialize_dx is removed too)